content/en/blog/_posts/2023-07-24-mixed-version-proxy-alpha.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Upgrade Kubernetes more safely by enabling Mixed Version Proxy (Alpha)"
+date: 2023-05-24
+slug: kubernetes-1-28-feature-mixed-version-proxy-alpha
+---
+
+**Author:** Richa Banker (Google)
+
+This blog describes “Mixed Version Proxy”, a new alpha feature in Kubernetes 1.28. Mixed Version Proxy enables a resource request to be served by the correct apiserver in cases where there are multiple apiservers at varied versions in a cluster, ex: during an upgrade of a cluster, when a runtime config changes and a rollout occurs.
```

Comment:
How about:
```suggestion
This blog describes the _mixed version proxy_, a new alpha feature in Kubernetes 1.28. The
mixed version proxy enables an HTTP request for a resource to be served by the correct API server
in cases where there are multiple API servers at varied versions in a cluster. For example,
this is useful during a cluster upgrade, or when you're rolling out the runtime configuration of
the cluster's control plane.
```

I like the conciseness. You are very welcome to use something else that is concise but expresses things differently.